index.html

```diff
@@ -1,248 +1,264 @@
+    <section>
+      <div class="about-me">
+        <h2 class="heading">About Me</h2>
+        <p class="description">
+          I can help you build a product,feature or website look through some
+          of my work and experience! If you like what you see and have a
+          project you need coded, don't hesitate to contact me.
+        </p>
+        <button>Get My Resume</button>
+      </div>
+    </section>
+    <!-- end of about me section -->
+
+    <!-- Language,framework & skills -->
+    <section class="abilities-section">
+      <div class="abilities">
+        <img src="./images/language.png" alt="language logo" />
+        <h3 class="heading-three">Languages</h3>
+        <ul class="language">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+
+      <div class="abilities">
+        <img src="./images/Frameworks.png" alt="framework logo" />
+        <h3 class="heading-three">Frameworks</h3>
+        <ul class="framework">
+          <li>Bootstrap</li>
+          <li>Ruby On Rails</li>
+          <li>Rspec</li>
+          <li>Selenium</li>
+          <li>Capybara</li>
+        </ul>
+      </div>
+
+      <div class="abilities">
+        <img src="./images/Skills.png" alt="skills logo" />
+        <h2 class="heading-three">Skills</h2>
+        <ul class="skills">
+          <li>Codekit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>GitLab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <!-- end of tech section -->
+
+    <!-- Contact Form -->
+    <section class="contact-container">
+      <h3 class="contact">
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h3>
+      <!-- forms -->
+      <form action="https://formspree.io/f/xoqbvjel" method="post">
+        <input type="text" placeholder="Full name" required maxlength="30" />
+        <br />
+        <input type="email" name="email " id="email" placeholder="Email address" required />
+        <br />
+        <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required
+          maxlength="500"></textarea>
+        <button type="submit">Get in touch</button>
+      </form>
+      <ul>
+        <li class="list-group">
+          <a href="https://@Maxwell_Chuks67"><img src="./images/Twitter.png" alt="Twitter Logo" /></a>
+          <a href="https://www.linkedin.com/in/chukwuemeka-orji-50b766241/"><img src="./images/Linkedin.png"
+              alt="Linkedin Logo" /></a>
+          <a href="https://github.com/Maxwell011"><img src="./images/Medium.png" alt="Medium Logo" /></a>
+          <a href="https://github.com/Maxwell011"><img src="./images/Vector.png" alt="GitHub Logo" /></a>
+          <a href="https://github.com/Maxwell011"><img src="./images/Angelist.png" alt="Angelist Logo" /></a>
+        </li>
+      </ul>
```

Comment:
- [ ] According to the project requirements, Flexbox must be used to layout the elements. Kindly make sure the ul > li content is aligned with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. 👍🏻